camera/CCameraAngles.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "../player/CPlayerAngles.hpp"
+
+class CCameraAngles
+{
+public:
+	char pad_0000[960]; //0x0000
+	CPlayerAngles* angles; //0x03C0
```

Comment:
Missing `m_` prefix, sorry.

Reply:
It's fine we everything is fixable, I just was too happy to merge.